app/src/main/java/com/mapbox/maps/testapp/examples/SecondaryDisplayPresentationActivity.kt

```diff
@@ -0,0 +1,201 @@
+package com.mapbox.maps.testapp.examples
+
+import android.app.Presentation
+import android.content.Context
+import android.content.DialogInterface
+import android.hardware.display.DisplayManager
+import android.os.Bundle
+import android.util.Log
+import android.view.Display
+import android.view.LayoutInflater
+import android.view.View
+import android.view.WindowManager
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.children
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LifecycleRegistry
+import androidx.lifecycle.ViewTreeLifecycleOwner
+import com.mapbox.maps.MapView
+import com.mapbox.maps.testapp.R
+import com.mapbox.maps.testapp.databinding.ActivitySecondaryDisplayPresentationBinding
+import com.mapbox.maps.testapp.databinding.ItemDisplayInfoBinding
+
+private const val TAG = "SecondaryDisplayPresent"
+
+private const val DISPLAY_INFO_TAG = "DisplayInfo"
```

Comment:
guess we're mostly defining consts in companion object in the bottom of the file